docs/content/sips/012-signing-spin-releases.md

```diff
@@ -0,0 +1,210 @@
+title = "SIP 012 - "Signing the Spin releases"
+template = "main"
+date = "2023-03-01T01:01:01Z"
+
+---
+
+Summary: This improvement proposal describes the reasoning and implementation
+for signing Spin releases.
+
+Owners: radu@fermyon.com
+
+Created: March 2, 2023
+
+## Background
+
+Signing release artifacts for software offers a way for consumers of the software
+to verify the integrity of the package they downloaded. The signature should
+offer two guarantees: the author of the artifact is indeed the one expected by
+the consumer, and the content of the artifact has not been tampered with since
+its creation.
+
+The goal of this SIP is to define the tools used for generating signatures for
+the Spin project, and to provide a standard process for signing release artifacts
+for Spin and related projects.
```

Comment:
Should we consider changing the way we expect plugins to be signed as well? Or are SHA checksums sufficient?